core/templates/dev/head/pages/exploration_editor/param_changes_editor_directive.html

```diff
@@ -47,9 +47,12 @@
       </span>
       <span>
         Change
-        <select2-dropdown item="paramChange.name" choices="paramNameChoices"
-                          placeholder="Param name" new-choice-regex="^[A-Za-z]+$"
-                          width="143px">
+        <select2-dropdown item="paramChange.name"
+                          choices="paramNameChoices"
+                          placeholder="Param name"
+                          new-choice-regex="^[A-Za-z]+$"
+                          width="143px"
+                          tag-mode="true">
```

Comment:
Okay,  I will take a look.

Okay, just that Parameterized Adventure exploration (adventure.yaml) from Admin tab contains parameters in TextInput and that confused me.